packages/test/test-end-to-end-tests/src/test/gc/gcSweepDataStores.spec.ts

```diff
@@ -224,7 +224,8 @@ const summarizationWithUnreferencedDataStoreAfterTime = async () => {
 		url: toFluidHandleInternal(testDataObject.handle).absolutePath,
 	});
 	const summarizerDataObject = response.value as ITestDataObject;
-	await delay(sweepTimeoutMs + 10);
+
+	clock.tick(sweepTimeoutMs + 10);
```

Comment:
We should be able to stop overriding the timeouts now, and just use the default values.  I think that would be preferable if it works (and good to know why if it doesn't)

Reply:
Good point. I changed it to use the default values. I had to re-arrange some logic but it worked. Session expiry is what makes it harder but I worked around it.